assignments/lambda-classes.js

```diff
@@ -1 +1,254 @@
+        super(attributes);
+        this.specialty = attributes.specialty;
+        this.favLanguage = attributes.favLanguage;
+        this.catchPhrase = attributes.catchPhrase;
+   
+  }
+        demo(subject) {
+            return console.log(`Today we are learning about ${subject}`);
+
+           }
+
+           grade(student,subject='PE') {
+            
+                return console.log(`${student.name} receives a perfect score on ${subject}`)
+ 
+               
+    
+           }
+        }
+
+
+  // #### Student
+
+// * Now we need some students!
+// * Student uses the same attributes that have been set up by Person
+// * Student has the following unique props:
+//   * `previousBackground` i.e. what the Student used to do before Lambda School
+//   * `className` i.e. CS132
+//   * `favSubjects`. i.e. an array of the student's favorite subjects ['Html', 'CSS', 'JavaScript']
+// * Student has the following methods:
+//   * `listsSubjects` a method that logs out all of the student's favoriteSubjects one by one.
+//   * `PRAssignment` a method that receives a subject as an argument and logs out that the `student.name has submitted a PR for {subject}`
+//   * `sprintChallenge` similar to PRAssignment but logs out `student.name has begun sprint challenge on {subject}`
+
+
+class Student extends Person {
+    constructor (attributes){
+        super(attributes);
+        this.previousBackground = attributes.previousBackground;
+        this.className = attributes.className;
+        this.favSubject = attributes.favSubject;
+   
+  }
+       listSubjects(favSubject) {
+           for(let i = 0; i < favSubject.length; i++) {
+               return console.log(favSubject[i]);
+           };
+       }
+
+       PRAssignment(subject) { 
+            return  console.log(`${this.name} has submitted a PR for ${subject}`);
+       }
+
+       sprintChallenge(subject) {
+            return console.log(`${this.name} has begun sprint challenge on ${subject}`)      
+
+  }};
+
+  
+
+// #### Project Manager
+
+// * Now that we have instructors and students, we'd be nowhere without our PM's
+// * ProjectManagers are extensions of Instructors
+// * ProjectManagers have the following unique props:
+//   * `gradClassName`: i.e. CS1
+//   * `favInstructor`: i.e. Sean
+// * ProjectManagers have the following Methods:
+//   * `standUp` a method that takes in a slack channel and logs `{name} announces to {channel}, @channel standy times!​​​​​
+//   * `debugsCode` a method that takes in a student object and a subject and logs out `{name} debugs {student.name}'s code on {subject}`
+
+    class ProjectManager extends Instructor {
+        constructor (attributes){
+        super(attributes);
+        this.gradClassName = attributes.gradClassName;
+        this.favInstructor = attributes.favInstructor;
+        
+     }
+       standUp (channel) {
+            return console.log(`${this.name} announces to ${channel}, @channel standy times!​​​​​`);
+            }
+
```

Comment:
Since you are console logging you don't need the return. I think we covered this when we talked about returning vs console logging in a function. If you are still confused just let me know :)